src/shared/components/case-history/case-history.component.spec.ts

```diff
@@ -120,12 +126,12 @@ describe('CaseHistoryComponent', () => {
           PlaceholderService,
           CaseReferencePipe,
           FormatTranslatorService,
-          { provide: AlertService, useValue: alertService },
-          { provide: ActivatedRoute, useValue: mockRoute },
-          { provide: OrderService, useValue: orderService },
-          { provide: CaseNotifier, useValue: caseNotifier },
-          { provide: CaseHistoryService, useValue: caseHistoryService },
-          { provide: Router, useValue: router }
+          {provide: AlertService, useValue: alertService},
+          {provide: ActivatedRoute, useValue: mockRoute},
+          {provide: OrderService, useValue: orderService},
+          {provide: CaseNotifier, useValue: caseNotifier},
+          {provide: CaseHistoryService, useValue: caseHistoryService},
+          {provide: Router, useValue: router}
```

Comment:
Not sure about formatting here - think spaces either side are better and the current consistent formatting but might have changed due to prettier?